tests/python_tests/test_gguf_reader.py

```diff
@@ -20,8 +20,6 @@
 @pytest.mark.parametrize("model_ids", get_gguf_model_list())
 @pytest.mark.precommit
 def test_pipelines_with_gguf_generate(pipeline_type, model_ids):
-    if sys.platform == 'darwin':
-        pytest.skip(reason="168882: Sporadic segmentation fault failure on MacOS.")
```

Comment:
JIRA ticket is still opened: https://jira.devtools.intel.com/browse/CVS-168882
Where is the fix?

Reply:
Looks the issue still exists

![Uploading {7E33C766-6E0D-42C1-BB1B-6A04D55F28FD}.png…]()

Reply:
Yes, as I mentioned above, this segment fault issue still exists, and not found the root cause yet, need your suggestions.